sql-statements/sql-statement-import-into.md

```diff
@@ -39,6 +33,9 @@ This TiDB statement is not applicable to TiDB Cloud.
 - The TiDB [temporary directory](/tidb-configuration-file.md#temp-dir-new-in-v630) is expected to have at least 90 GiB of available space. It is recommended to allocate storage space that is equal to or greater than the volume of data to be imported.
 - One import job supports importing data into one target table only. To import data into multiple target tables, after the import for a target table is completed, you need to create a new job for the next target table.
 - `IMPORT INTO` is not supported during TiDB cluster upgrades.
+- When the [Global Sort](/tidb-global-sort.md) feature is used for data import, the data size of a single row after encoding cannot exceed 32 MiB.
+- When the Global Sort feature is used for data import, if the target TiDB cluster is deleted before the import task is completed, temporary data used for global sorting might remain on Amazon S3. In this case, you need delete the residual data manually to avoid increasing S3 storage costs.
```

Comment:
```suggestion
- When the Global Sort feature is used for data import, if the target TiDB cluster is deleted before the import task is completed, temporary data used for global sorting might remain on Amazon S3. In this case, you need to delete the residual data manually to avoid increasing S3 storage costs.
```